sim/firesim-lib/src/main/cc/bridges/tracerv.cc

```diff
@@ -126,31 +133,38 @@ tracerv_t::tracerv_t(
             fprintf(stderr, "Invalid trace format arg\n");
         }
     } else {
-        fprintf(stderr, "TraceRV: Warning: No +tracefileN given!\n");
+        fprintf(stderr, "TraceRV %d: Tracing disabled, since +tracefile was not provided.\n", tracerno);
+        this->trace_enabled = false;
     }
 
     if (fireperf) {
+        assert(false && "FirePerf support temporarily disabled pending multiclock bringup");
```

Comment:
this still works, right?

Reply:
I haven't actually tested it, i just did all the work that seemed immediately necessary. 

How do i test fireperf?

Reply:
This should work: https://docs.fires.im/en/latest/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV-with-FlameGraph.html
to generate a flamegraph of Linux booting

I don't see any reason why fireperf shouldn't just work with this new setup.